### PR TITLE
Add more docs about cairo native within the code

### PR DESCRIPTION
**Pull request by edg-l**
## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

File: src/execution_result.rs [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/// # Execution Result
+///
+/// This module contains the structures used to interprete the program execution results, either
```
Comment (juanbono) [inline src/execution_result.rs:3]: typo: interprete -> interpret

**Review by juanbono — APPROVED** [review]

**Review by juanbono — CHANGES_REQUESTED** [review]
check typos

**Review by azteca1998 — DISMISSED** [review]

**Review by juanbono — APPROVED** [review]

**Comment by juanbono** [comment]
Please fix conflicts @edg-l

**Comment by edg-l** [comment]
> Please fix conflicts @edg-l

done

**Review by juanbono — APPROVED** [review]

**Pull request merged**